verify_rpms/rpm_verifier.py

```diff
@@ -44,85 +47,83 @@ def get_rpmdb(container_image: str, target_dir: Path, runner: Callable = run) ->
     return target_dir
 
 
-def get_unsigned_rpms(rpmdb: Path, runner: Callable = run) -> list[str]:
-    """
-    Get all unsigned RPMs from RPM DB path
-    Filter and return the unsigned RPMs
-    :param rpmdb: path to RPM DB folder
-    :param runner: subprocess.run to run CLI commands
-    :return: list of unsigned RPMs within the folder
-    """
```

Comment:
Can we keep the parts of the documentation which is still relevant?

Reply:
We have it in the `get_unsigned_rpms` method, it was moved

Reply:
See line 68

Reply:
Can we have something similar for `get_rpms_data` as well?

Reply:
Done